.jenkins

```diff
@@ -221,7 +221,7 @@ pipeline {
                                     -D CMAKE_CXX_FLAGS="-Wpedantic -Wall -Wextra" \
                                     -D CMAKE_CXX_CLANG_TIDY="$LLVM_DIR/bin/clang-tidy" \
                                     -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$BOOST_DIR;$BENCHMARK_DIR" \
-                                    -D ARBORX_ENABLE_MPI=ON \
+                                    -D ARBORX_ENABLE_MPI=OFF \
```

Comment:
This would disable clang-tidy checks on code behind `ARBORX_ENABLE_MPI` guards

Reply:
Ah, good point too. Did not notice `clang-tidy` is only for `Clang`. `CUDA-10.0-Clang` then, maybe?